backend/routes/readme.md

```diff
@@ -0,0 +1,9 @@
+Full readme and front end tests can be found here - still under construction:
+
+https://github.com/spicycheeto/XRPL-NFT-Proof-of-Attendance-Infrastructure

```

Comment:
could you change this? since we are moving to xrpl bounties repo

Reply:
Updated all readme files.